node-repository/src/main/java/com/yahoo/vespa/hosted/provision/NodeList.java

```diff
@@ -179,6 +180,13 @@ public NodeList group(int index) {
                                                        .findFirst());
     }
 
+    public ClusterResources toResources() {
+        if (isEmpty()) return new ClusterResources(0, 0, NodeResources.unspecified());
+        return new ClusterResources(size(),
+                                    (int)stream().distinct().count(),
```

Comment:
Shouldn't there be a map to to group id here?

Reply:
Yes, thanks!